tutorials/tutorial-batch-processing/tutorial-batch.md

```diff
@@ -0,0 +1,98 @@
+# Batch processing
+
+![Open batch button position](./images/open_batch.png)
+
+To open window for batch processing use marked button or use *Batch processing* position in *File* menu
+
+## Create a workflow for a batch processing
+
+![Prepare plan view](./images/prepare_plan.png)
+
+1.  Select how the mask file for segmentation will be recognized. 
+
+   a. In section *Use mask from*, Select *File* tab. 
+   
+   b. Select *Suffix*. 
+   
+   c. Input “_mask” in  *Use suffix* field. (All masks created by the *ROI Mask* component of Partseg have `_mask` suffix)
+   
+   d. Confirm with the "Add mask" button 
+   
+   e. File mask is now added to the workflow 
+
+2.  In *Roi Extraction* section
+ 
+   a. Select profile “chromosome1_prof” created in [chromosome 1 segmentation tutorial](../tutorial-chromosome-1/tutorial-chromosome1_16.md)
+   
+   b. All settings used in the selected profile are visible in the *information* section.
+   
+   c. Confirm with the "Add Profile/Pipeline" button. 
+   
+   d. The segmentation profile is now added to the workflow.
+
+3.  in section *Set of measurements*:
+
+   a. Select set “test_case” created in [chromosome 1 segmentation tutorial](../tutorial-chromosome-1/tutorial-chromosome1_16.md).
+   
+   b. All measurements within the selected set are visible in the *information* section
+    
+   c.  Select channel (x) and units (um) that will be used in quantification.
+   
+   d.  Confirm with the "Add Set of measurements" button 
+   
+   e.  The segmentation profile is now added to the workflow (3c).
+
+4.  Save created Workflow under the chosen name (e.g. *Chromosome1_batch*) with the "Save" button.
+
+5.  A new workflow is now added to a list of workflows.
+ 
+Note: to change any element of the existing workflow check box *Update element* (**U**) in the workflow edition mode. 
+Now select each of the elements that need replacement and update using the *Replace* buttons (**R**) in each section.
+![Prepare plan view note](./images/prepare_plan2.png) 
+
+## Execute created workflow
+
+![run workflow view](./images/input_files.png) 
+
+1.  Select list of files to be proceed
+   
+   a. using “Select directory” option select directory where files are present and add `/*[0-9].tif` to the path line (see [explanation](https://en.wikipedia.org/wiki/Glob_(programming))) Files created by the Mask Segmentation module of Partseg are saved as tifs, which names composed of the name of the original 3D microscopic picture and component number at the end. 
+   
+   b. Push "Find All" button to finds all files matching pattern
+   
+   c. confirm by the "Add" button.
+   
+   ![Preview of found files](./images/add_files.png) 
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 1 space](https://app.codacy.com/manual/Czaki/PartSeg/pullRequest?prid=6358262)